sysutils/lima/Portfile

```diff
@@ -22,44 +29,40 @@ maintainers         {gmail.com:herby.gillot @herbygillot} \
                     openmaintainer
 
 depends_run         port:qemu
+depends_build-append \
+                    port:sudo
```

Comment:
Can you describe why `sudo` needed to be added to the build phase?

Reply:
Ah, sorry — that was a leftover from an earlier attempt at dropping `root` privileges; removed!